independent-publisher-connectors/Care Quaility Comission For England/CQC-Connector.swagger.json

```diff
@@ -0,0 +1,843 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CQC Data",
+    "description": "To retrieve regulator review reports and ratings for England",
+    "version": "10",
+    "contact": {
+      "name": "Martyn Lesbirel",
+      "email": "ipc-author-cqc@outlook.com"
+    }
+  },
+  "host": "apicqcorguk",
+  "basePath": "/public/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/providers": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Providers",
```

Comment:
@martynlesbirel Can you please provide a valid summary value as it would be visible to public.

Documentation - [Summary values](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the similar summary values in the swagger

Reply:
Summary values revised and size verified as acceptable via paconn